automated/linux/pibridge/test-pb-1.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
```

Comment:
this script is somehow the testing of DIO module, not prbridge, isn't it?

Reply:
I would say, that this tests the pibridge communication indirectly by switching some IOs and checking the values on another IO-module. For me it's ok and it shows if the communication with the modules is working (a piTest -d should theoretically enough). Feel free to make suggestions for improvement. ;)

Reply:
Oh, forgot to continue on this thread.  I think we can make it in 2 layer, the script is targeted at functions and testcase ymal data focus on test purpose. e.g.   a script (named test_dio.sh maybe) just to focus on the testing of DIO(DI,DO included), a testcase ymal to test all pins on DIO (further testcase for DI, DO, etc) with test_dio.sh, another testcase ymal to test the pibridge but also with test_dio.sh.